src/mac.cpp

```diff
@@ -278,9 +278,12 @@ void FocusApplication(const std::string& application) {
       if ([app.bundleURL.path.lowercaseString containsString:name]) {
         [app unhide];
         [app activateWithOptions:NSApplicationActivateIgnoringOtherApps];
+        CFRelease(windows);
+        return;
```

Comment:
Was this added just to break out of the for-loop?

Reply:
yep, if multiple applications matched the same string, it would focus them in quick succession. this now just takes the first one.

Reply:
oh sorry, now I see what you're saying, we can just use a break here. will update.